aws/provider_test.go

```diff
@@ -748,6 +748,26 @@ func testAccOrganizationsEnabledPreCheck(t *testing.T) {
 	}
 }
 
+func testAccOrganizationsAWSServiceAccessRamEnabledPreCheck(t *testing.T) {
```

Comment:
I tried using `aws_ram_resource_share_accepter` instead but it does not work as I described in https://github.com/hashicorp/terraform-provider-aws/issues/16578. So I needed to enable the service principal instead to write the acc test.